src/index.scss

```
@@ -8,351 +8,357 @@ tablet has fixed widths and font sizes
+		height: 150px;
+		margin-left: 40px;
+	}
+
+	@include mobile {
+		/*68 px is the current height of the nav bar
+		will think of a better way to structure this later*/
+		width: default;
+		height: max(68px, calc(35px + 10vh));
+		margin-left: 15px;
+	}
 }
 
 .navigation {
-  width: 100%;
-
-  font-size: 15px;
-  font-weight: 500;
-
-  color: $blue;
-  background-color: white;
-
-  ul {
-    list-style-type: none;
-    float: right;
-
-    margin: 0;
-    padding: 0;
-
-    li {
-      padding: 25px 25px;
-      text-align: center;
-      text-decoration: none;
-
-      a {
-        color: $blue;
-
-        border-bottom-style: solid;
-        border-color: transparentize($blue, 1);
-        transition: border-color 0.2s;
-
-        &:hover {
-          border-color: transparentize($blue, 0);
-          transition: border-color 0.2s;
-        }
-      }
-    }
-  }
-
-  @include tablet {
-    display: inline-block;
-    
-    font-size: 12px;
-    font-weight: 400;
-
-    color: white;
-    background-color: $blue;
-
-    ul li {
-      padding: 25px 15px;
-      a {
-        color: white;
-        border-bottom-style: solid;
-        border-color: transparentize(white, 1);
-
-        &:hover {
-          border-color: transparentize(white, 1);
-        }
-        &:active {
-          border-color: transparentize(white, 0);
-        }
-      }
-    }
-  }
-
-  @include mobile {
-    /*no scaling for very short devices,
-    scaling matching the mlh badge scaling for avg devices,
-    high scaling for very tall devices
-    */
-
-    margin-bottom: max(1px, calc(10vh - 40px), calc(30vh - 190px));
-  }
+	width: 100%;
+
+	font-size: 15px;
+	font-weight: 500;
+
+	color: $blue;
+	background-color: white;
+
+	ul {
+		list-style-type: none;
+		float: right;
+
+		margin: 0;
+		padding: 0;
+
+		li {
+			padding: 25px 25px;
+			text-align: center;
+			text-decoration: none;
+
+			a {
+				color: $blue;
+
+				border-bottom-style: solid;
+				border-color: transparentize($blue, 1);
+				transition: border-color 0.2s;
+
+				&:hover {
+					border-color: transparentize($blue, 0);
+					transition: border-color 0.2s;
+				}
+			}
+		}
+	}
+
+	@include tablet {
+		display: inline-block;
+		
+		font-size: 12px;
+		font-weight: 400;
+
+		color: white;
+		background-color: $blue;
+
+		ul li {
+			padding: 25px 15px;
+			a {
+				color: white;
+				border-bottom-style: solid;
+				border-color: transparentize(white, 1);
+
+				&:hover {
+					border-color: transparentize(white, 1);
+				}
+				&:active {
+					border-color: transparentize(white, 0);
+				}
+			}
+		}
+	}
+
+	@include mobile {
+		/*no scaling for very short devices,
+		scaling matching the mlh badge scaling for avg devices,
+		high scaling for very tall devices
+		*/
+
+		margin-bottom: max(1px, calc(10vh - 40px), calc(30vh - 190px));
+	}
 }
 
 .section {
-  height: 100%;
-  display: flex;
-  flex-direction: column;
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+	@include desktop{
```

```suggestion
	@include desktop {
```
Nit: but while we're in here!